cli/Cargo.toml

```diff
@@ -23,20 +23,20 @@ name = "seth"
 path = "src/main.rs"
 
 [dependencies]
-clap = "2.32"
+clap = "2"
 config = "0.9"
-dirs = "1.0"
+dirs = "2.0"
```

Comment:
Why not remove the point release and do 'dirs = "2"' and similar throughout. For 0.x we need both but for 1+, just the major should be fine, if projects conform to semver.

Reply:
Updated